byzcoin/ringbuf.go

```diff
@@ -0,0 +1,45 @@
+package byzcoin
+
```

Comment:
I know it's not a common practice in this repo, but can you add few sentences describing why this file is needed? (I have a bad memory, and I know that I will soon wonder why this file is here...)